test/complex.jl

```diff
@@ -489,8 +486,8 @@ end
 end
 
 @testset "tanh" begin
-    #  tanh(conj(z)) = conj(tanh(z))
-    #  tanh(-z) = -tanh(z)
+    # tanh(conj(z)) = conj(tanh(z))
+    # tanh(-z) = -tanh(z)
```

Comment:
Likewise here, perhaps incorporate the comments in the testset description and nix the comments?